doc/sphinx/source/data/commondata.rst

```diff
@@ -0,0 +1,355 @@
+If an update requires to change the number of datapoint,
+it should be added as a separate observable.
+
+``observable``
+~~~~~~~~~~~~~~
+This is a dictionary with the entries ``description``, ``label`` and ``units``.
+All entries must be latex-compilable as they are used by various plotting routines in ``validphys``.
+
+``kinematics::file``
+~~~~~~~~~~~~~~~~~~~~
+A reference to a ``.yaml`` file containing all kinematic information.
+The file contain a list of ``ndata`` ``bins`` for which information about all variables
+is included for all bins.
+When ``mid`` is not given, it will be automatically filled with the midpoint between min and max.
+Only ``mid`` is used for cuts, while ``min`` and ``max`` may be used for plotting routines.
+
+..  code-block:: yaml
+
+    bins:
+        - var_1:
+            min: 0
+            max: 1
+            mid: 0.5
+          var_2:
+            min: 0
+            max: 1
+            mid: 0.5
+
+``kinematics::variables``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Metadata for each of the variables contained in the ``kinematics::file``
+and which can be ``description``, ``label`` and ``units``.
+Latex syntax is accepted and encouraged since they will be used by plotting routines.
+
+..  code-block:: yaml
+
+    variables:
+      var_1: {description: "my var 1", label: "$m$", "units: "GeV"}
+
+
+``kinematic_coverage``
+~~~~~~~~~~~~~~~~~~~~~~
+A list of the variables within the kinematic files
+
+
+``data_central``
+~~~~~~~~~~~~~~~~
+A reference to a ``yaml`` file containing the measurement central data.
+The format of the data is a ``yaml`` file with an entry ``data_central`` which
+list for all values for all bins.
+
+..  code-block:: yaml
+
+    data_central:
+        - val1
+        - val2
+        - val3
+
+``data_uncertainties``
+~~~~~~~~~~~~~~~~~~~~~~
+A list of ``.yaml`` file containing the uncertainty information for the measurement.
+When using more than one uncertainty file they will be concatenated. 
+This allows the user the flexibility of creating variants
+where only a subset of the uncertainties are modified.
+
+The format of the uncertainty files is of two fields, a ``definitions`` field that contains
+metadata about all the uncertainties: name, treatment (``ADD`` or ``MULT``) and type
+and a second field ``bins`` which is a list of mappings with ``ndata`` entries
+with the named uncertainties.
+
+Note that, regardless of their treatment, uncertainties should always be written as absolute values
+and not relative to the data values. If the data should be updated, the uncertainties should be too.
+
+..  code-block:: yaml
+
+    definitions:
+        stat:
+            description:
+            treatment:
+            type:
+        error_name:
+            description:
+            treatment:
+            type:
+        error_name_2:
+            description:
+            treatment:
+            type:
+    bins:
+        - stat:
+          error_name:
+          error_name_2:
```

Comment:
```suggestion
    definitions:
        stat:
            description: statistical error
            treatment: ADD
            type:
        error_name:
            description: an additive uncertainty
            treatment: ADD
            type:
        error_name_2:
            description: an multiplicative uncertainty
            treatment: MULT
            type:
    bins:
        - stat: 1.0
          error_name: 2.0
          error_name_2: 3.0
```
what is `type`? it is not mentioned above ...

Reply:
The type. A lumi uncertainty, calorimeter, etc.

(i agree someone should define it better...)